discord_signup.py

```diff
@@ -0,0 +1,204 @@
+# This is a test file for a user sign-up flow.
+# It requires the bot to be alive at all times, so I have not merged it into the main script (yet?)
```

Comment:
correct, I don't think that a not can be hosted on two separate machines simultaneously, so having this one on at all times would cause the daily run to fail